### PR TITLE
[FM v0.5.0]--Add doc release note

### PR DESCRIPTION
**Pull request by Huanli-Meng**
### Motivation

Add a doc release note about Function Mesh v0.5.0

### CODE REVIEW

File: docs/releases/release-note-0-5-0.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Set function log levels
+
+In this release, Function Mesh supports setting multiple [log levels](/functions/function-crd.md#log-levels) for different Pulsar function runtimes. By default, the log level for Pulsar functions is `info`. You can use the `spec.log.level` option to specify a particular log level. Or, you can customize a log configuration file based on your requirements and reference to the log configuration file by using the `spec.log.logConfig` option.
+For details, see [set log levels](/functions/produce-function-log.md#set-log-levels).
+
+## Build function images using Buildpacks
+
+To deploy a function to a Pulsar cluster, you need to package the function into an artifact or build the function into a Docker image.
+
+Starting from this release, you can use [Buildpacks](https://buildpacks.io/docs/concepts/components/buildpack/) to build a function image through multiple buildpacks. Buildpacks helps customize your codes as required, change the runner base (`pulsar-function-<runtime>-runner`) of a function image, and switch the runner image versions without building the function again.
```
Comment (teachwrite) [inline docs/releases/release-note-0-5-0.md:18]:
```suggestion
Starting from this release, you can use [Buildpacks](https://buildpacks.io/docs/concepts/components/buildpack/) to build a function image through multiple buildpacks. Buildpacks help customize your code as required, change the runner base (`pulsar-function-<runtime>-runner`) of a function image, and switch the runner image versions without building the function again.
```

File: docs/releases/release-note-0-5-0.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+For details about how to build function images using Buildpacks, see Function Mesh documentation.
```
Comment (teachwrite) [inline docs/releases/release-note-0-5-0.md:20]:
```suggestion
For details about how to build function images using Buildpacks, see the Function Mesh documentation.
```

**Review by teachwrite — APPROVED** [review]
Just a couple small changes. Otherwise, it LGTM.

**Review by danpi — APPROVED** [review]
LGTM.

**Review by danpi — APPROVED** [review]

**Pull request merged**